Doc/glossary.rst

```diff
@@ -367,16 +378,18 @@ Glossary
       and the :ref:`function` section.
 
    function annotation
-      An arbitrary metadata value associated with a function parameter or return
-      value. Its syntax is explained in section :ref:`function`.  Annotations
-      may be accessed via the :attr:`__annotations__` special attribute of a
-      function object.
+      An :term:`annotation` of a function or method.
+
+      For example, this function is annotated as requiring its parameters to be
+      int and as returning an int as well::
```

Comment:
"int" -> "an int" at beginning of this line.

Reply:
Didn't use "an int" there because we are talking about two parameters, in plural. Anyway, I changed this lines a little, I hope it makes it clearer.